lib/mpl_toolkits/axes_grid1/inset_locator.py

```diff
@@ -312,7 +312,8 @@ def __init__(self, bbox1, bbox2, loc1, loc2=None, **kwargs):
             raise ValueError("transform should not be set")
 
         kwargs["transform"] = IdentityTransform()
-        Patch.__init__(self, fill=False, **kwargs)
+        fill = 'fc' in kwargs
```

Comment:
This needs to also check for 'facecolor' and 'color'.  Use `cbook.normalize_kwargs` to sort out the aliases.